pkg/translator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -24,6 +24,7 @@ type Settings struct {
 	ExportCreatedMetric bool
 	AddMetricSuffixes   bool
 	SendMetadata        bool
+	RetryOnHTTP429      bool
```

Comment:
This should be removed, no?  The translator has nothing to do with whether the PRW request is retried.

Reply:
Removed,